Remotion/Web/Core/Res/Themes/NovaViso/HTML/Common.css

```diff
@@ -0,0 +1,109 @@
+:root {
+    /*colors*/
+    --primary: #0C6594;
+    --highlight: #DEF3FE;
+    --window-border: #C4C4C4;
+    --separator: #E1E1E1;
+    --menu-background: #F9F9F9;
+    --hover: #218FC9;
+    --border: #979797;
+    --dossier-section-yellow: #DFAF00;
+    --dossier-section-red: #DA2222;
+    --dossier-section-green: #4AD439;
+    --focus: #23ADF8;
```

Comment:
please check focus, the style guide seems to use primary for focus?

Reply:
I'm not sure what I was thinking about originally, but we probably want to declare the colors only once and then use those color properties when we define focus, etc. It may be easier/faster to verify the correct colors if we reuse them now, and we can still split them up later should be learn more.